composer.json

```diff
@@ -5,7 +5,10 @@
     "prefer-stable": true,
     "config": {
         "preferred-install": "dist",
-        "secure-http": false
+        "secure-http": false,
+        "platform": {
+            "ext-mongo": "1.6"
```

Comment:
It's a pity any mongo related stuff is needed for development, especially for those not using `MessagesWidget` (docs claim it uses mongo).

Anyway, `ext-mongo` is being deprecated in favour of `ext-mongodb` and that's what I tried to use at first. Unfortunately, `doctrine/doctrine-mongo-odm-module` (actually `doctrine/mongodb` as a dep) [seems stuck](https://github.com/doctrine/mongodb/issues/239) with `ext-mongo`.

The consequences for us are:
- we cannot use php7 during development
- [MongoDB 3.1.x is going to be the last version compatible with rtm](https://docs.mongodb.com/ecosystem/drivers/php/#mongodb-compatibility)

Note, this doesn't change anything in deployment process, as we cap4all. It resolves composer deps on the machine you deploy from and simply send a zip ball to the target machine. So overall, we don't just composer platform packages to check production dependency at all.